.github/mkosi.pkgmngr/gentoo/etc/portage/binrepos.conf

```diff
@@ -0,0 +1,2 @@
+[binhost]
+sync-uri = https://raw.githubusercontent.com/257/binpkgs/master
```

Comment:
Why bother? We already set `PORTAGE_BINHOST` anyway

Reply:
dropped, never mind.